### DISCUSSION TITLE
Appium WebDriveragent project showing missing dependencies marked in red

### DISCUSSION THREAD

**Discussion by Ancybaby** [appium/.github#7] [Q&A]
Hi ,
When I open the Appium-webdriveragent project (version 4.9.0 ) in XCode (version 15.4), it shows red marking for Resources , WebdriverAgentLib and WebDriverAgentRunner. I cannot see the Resources folder in finder too.  Can you please help me out in this ? I'am new to the Appium server setup. Any help would be great.  Thanks

**Comment by Ancybaby** [comment]
<img width="1462" alt="Screenshot 2024-07-15 at 4 24 01 PM" src="https://github.com/user-attachments/assets/5904699b-2db2-4021-b8b2-e0caac66f370">

<img width="1462" alt="Screenshot 2024-07-15 at 4 27 17 PM" src="https://github.com/user-attachments/assets/048fea19-3a7f-45e9-9c93-1ff8333963c6">

**Reply by KazuCocoa** [reply]
The error could indicate you installed the modules for another user, for example. Perhaps you built the environment with your own custom users. Then, firstly, we'd recommend you prepare the environment with the currently logged-in user.
Asking such a general question (this permission stuff is not Appium specific) in https://discuss.appium.io/ would also help.
